build.gradle.kts

```
@@ -98,6 +104,7 @@ tasks {
         this.archiveClassifier.set(null as String?)
         this.archiveFileName.set("${project.name}-${project.version}.${this.archiveExtension.getOrElse("jar")}")
         this.destinationDirectory.set(rootProject.tasks.shadowJar.get().destinationDirectory.get())
+        relocate("cloud.commandframework", "com.thevoxelbox.voxelsniper.cloud")
         relocate("org.incendo.serverlib", "com.thevoxelbox.voxelsniper.serverlib")
```

```suggestion
        relocate("io.leangen.geantyref", "com.thevoxelbox.geantyref")
```
That dependency is added by cloud.